### PR TITLE
plan: add tests of flag for FieldType

### PR DESCRIPTION
**Pull request by XuHuaiyu**
We didn't check the flag of FieldType before, this commit adds some tests to check this.
PTAL @shenli @coocood

### CODE REVIEW

**Comment by coocood** [comment]
LGTM
Good job!

File: plan/physical_plan_test.go [hunk 1]
```diff
@@ -54,14 +54,14 @@ func (s *testPlanSuite) TestPushDownAggregation(c *C) {
 			sql:       "select sum(b) from t group by c",
 			best:      "Table(t)->HashAgg->Projection",
 			aggFuns:   "[sum(test.t.b)]",
-			aggFields: "[blob decimal]",
+			aggFields: "[blob decimal BINARY]",
```
Comment (shenli) [inline plan/physical_plan_test.go]: Please make sure you can pass tidb-test.

**Comment by shenli** [comment]
@XuHuaiyu Please resolve the conflicts.

File: plan/physical_plan_test.go (hunk 1 above)
Comment (XuHuaiyu) [inline plan/physical_plan_test.go]: ok

Comment (XuHuaiyu) [inline plan/physical_plan_test.go]: ok

**Comment by XuHuaiyu** [comment]
@shenli 
This commit can pass tidb-test. PTAL

**Comment by XuHuaiyu** [comment]
@shenli 
This commit can pass tidb-test. PTAL

**Pull request merged**